frontend/apps/reader/modules/readerhandmade.lua

```diff
@@ -552,6 +555,7 @@ function ReaderHandMade:isInHiddenFlow(pageno)
 end
 
 function ReaderHandMade:toggleHiddenFlow(pageno)
+    self.ui.annotation:setNeedsUpdateFlag()
     local idx, is_match = self:_getItemIndex(self.flow_points, pageno)
```

Comment:
Any idea why this one was needed here - while a `self.view.footer:onUpdateFooter` is not ? In your other 3 additions of this, they seem to go together.
(Not really familiar any longer with ReaderHandmade, and not really able to get a better overview of it right now - but it feels adding external flagging with self.ui.annotations... in this quite internal toggleHiddenFlow may be odd.)

Reply:
`toggleHiddenFlow()` is called in the Page browser only (long-press on a page thumbnail).
The footer is updated on closing the Page browser.